modules/claims_api/spec/requests/v2/power_of_attorney_requests/decisions/create/request_spec.rb

```diff
@@ -3,7 +3,9 @@
 require 'rails_helper'
 require Rails.root / 'modules/claims_api/spec/rails_helper'
 
-RSpec.describe 'ClaimsApi::V2::PowerOfAttorneyRequests::Decisions#create', :bgs, type: :request do
+# TODO: Delete this file
+# We are covered by request_controller_spec.rb
+RSpec.describe 'ClaimsApi::V2::PowerOfAttorneyRequests::Decisions#create', :bgs, :skip, type: :request do
```

Comment:
this due to LoC?

Reply:
Yes. I wanted to delete this file and move/rename the other files to align with the API and code directories, but ran into the line limit bot 🤖 😔